server/container/guice/protocols/jmap/src/main/java/org/apache/james/jmap/rfc8621/RFC8621MethodsModule.java

```diff
@@ -103,6 +105,8 @@ protected void configure() {
         methods.addBinding().to(ThreadGetMethod.class);
         methods.addBinding().to(VacationResponseGetMethod.class);
         methods.addBinding().to(VacationResponseSetMethod.class);
+        methods.addBinding().to(MDNParseMethod.class);
```

Comment:
MDNParse method is already added line 103.

Also please respect alphabetic ordering.